locales/en/moves.json

```diff
@@ -120,5 +120,8 @@
   },
   "agreement_status": {
     "heading": "Has this move been agreed with receiving prison?"
+  },
+  "date_range": {
```

Comment:
Sorry, I didn't spot this earlier, but can we move this to the same place as the other step headings please - nested under `steps`. That way it will make them logically grouped and easier to find in the future.